app/components/e-heading.md

```diff
@@ -0,0 +1,51 @@
+### Headings
+
+```
+<div>
+  <e-heading tagName="h1" :underline="false">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading>
+  <e-heading tagName="h2" :underline="false">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading>
+  <e-heading tagName="h3" :underline="false">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading>
+  <e-heading tagName="h4" :underline="false">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading>
+  <e-heading tagName="h5" :underline="false">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading>
+  <a href="#"><e-heading tagName="h6" :underline="false">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading></a>
+</div>
+```
+
+### Headings with alternative colors
+
+```
+<div>
+  <e-heading tagName="h1" color="gray">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading>
+  <e-heading tagName="h2" color="green">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading>
+  <e-heading tagName="h3" color="gray">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading>
+  <e-heading tagName="h4" color="green">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading>
+  <e-heading tagName="h5" color="blue">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading>
+  <a href="#"><e-heading tagName="h6" color="green">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading></a>
+</div>
+```
+
+### Headings with underline
+
+```
+<div>
+  <e-heading tagName="h1" :underline="true">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading>
+  <e-heading tagName="h2" :underline="true">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading>
+  <e-heading tagName="h3" :underline="true">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading>
+  <e-heading tagName="h4" :underline="true">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading>
+  <e-heading tagName="h5" :underline="true">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading>
+  <a href="#"><e-heading tagName="h6" :underline="true">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading></a>
+</div>
+```
+
+### Headings with alternative font weights
+
+```
+<div>
+  <e-heading tagName="h1" weight="bold">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor</e-heading>
```

Comment:
- [ ] In this case please replace `Lorem ipsum` with text that describes the font-weight. Else it's not clear what you want to show here without looking at the code.